Update 2025-02-10.md (#754)
elevenlabs/elevenlabs-docs: fern/docs/pages/changelog/2025-02-10.md

## Conversational AI

> - **Updated Pricing**: Updated the pricing for Conversational AI with a new [self-serve free tier](/docs/conversational-ai/overview#pricing-tiers).

Reviewer: its not just free tier, i would just say with new self-serve pricing . we reduced prices by up to 50% with new self-serve pricing tiers

> - **Updated Pricing**: Updated the pricing for Conversational AI with a new [self-serve free tier](/docs/conversational-ai/overview#pricing-tiers).

Author: ```suggestion
- **Updated Pricing**: Updated self-serve pricing for Conversational AI with [reduced cost and a free tier](/docs/conversational-ai/overview#pricing-tiers).
```
